pkg/secrethub/client.go

```diff
@@ -80,6 +84,7 @@ func NewClient(with ...ClientOption) (*Client, error) {
 	client := &Client{
 		httpClient:    http.NewClient(),
 		repoIndexKeys: make(map[api.RepoPath]*crypto.SymmetricKey),
+		ConfigDir:     &configdir.Dir{},
```

Comment:
Should this be initialized to `nil`? So that the nil check on line 96 can set it?

Reply:
![](https://media3.giphy.com/media/Kxz4XFFihiInUclAnc/giphy.gif)